modules/core/utils/upgrade_nomad_clients.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/bash
+set -euo pipefail
+
+read -p 'AWS_PROFILE: ' awsProfile
+read -p 'NOMAD_ADDR: ' nomadAddr
+read -p 'NOMAD_TOKEN: ' nomadToken
+read -p 'ASGName: ' ASGName
```

Comment:
This is the only argument that's "required". The rest are basically connection options that can be passed alongside environment variables like I mentioned before.

Parse commands with something like I did here: https://github.com/GovTechSG/terraform-modules/blob/master/modules/core/packer/roles/nomad/files/configure.sh